_shared/hooks/post_gen_project.py

```diff
@@ -141,7 +141,6 @@ def main():
             "{{ cookiecutter.package_name }}/__init__.py",
             "{{ cookiecutter.package_name }}/app.py",
             "Dockerfile",
-            "docker-compose.yml",
```

Comment:
This removes `docker-compose.yml` from `template_ignore_patterns` which means that `make template` will now update `docker-compose.yml`. (On `main` the cookiecutter will generate the initial `docker-compose.yml` file when creating a new project but `make services` will never update `docker-compose.yml`).

If a project doesn't _want_ `make template` to change `docker-compose.yml` it can just add `docker-compose.yml` to its `ignore` list in `cookiecutter.json`.